recipes/khiopsdriver-gcs/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Set-up the shell to behave more like a general-purpose programming language
+set -euo pipefail
+
```

Comment:
```suggestion
export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"

```
Please add this variable instead as you likely only need to disable the symbol availability check rather than requiring a newer `c_stdlib`.